### PR TITLE
Feature: Salesforce Support Daily Logs (48)

### PR DESCRIPTION
**Pull request by vg-svitla**

### CODE REVIEW

File: Salesforce/trigger_salesforce_events.json [hunk 1]
```diff
@@ -22,6 +22,11 @@
         "type": "integer",
         "description": "Batch frequency in seconds",
         "default": 600
+      },
```
Comment (squioc) [inline Salesforce/trigger_salesforce_events.json:25]: Could you update [connector_salesforce_events.json](https://github.com/SEKOIA-IO/automation-library/blob/main/Salesforce/connector_salesforce_events.json) also?

**Review by squioc — CHANGES_REQUESTED** [review]
Code looks good to me.
Could you update the connector manifest file? Thanks.

File: Salesforce/trigger_salesforce_events.json (hunk 1 above)
Comment (vg-svitla) [reply Salesforce/trigger_salesforce_events.json:25]: Along with `fetch_daily_logs` I also added `frequency` to `connector.json` to make them sync. It is fine?

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**